Krypt/Source/pkcs8.c

```diff
@@ -32,3 +37,57 @@ char *convert_pkcs1_to_pkcs8(const char *pem) {
     
     return str_from_BIO(out);
 }
+
+char *pkcs8_encrypt(const char *pkcs1, const char *password) {
+  BIO *out = NULL;
+  EVP_PKEY *pkey = NULL;
+  PKCS8_PRIV_KEY_INFO *p8inf = NULL;
+  X509_ALGOR *pbe = NULL;
+  X509_SIG *p8 = NULL;
+
+  const EVP_CIPHER *cipher = EVP_aes_256_cbc();
```

Comment:
do you need to free this pointer in case of failure as well?

Reply:
Couldn't find any `EVP_CIPHER_free` function, also is not cleared in the apps the OpenSSL ships so don't think it's necessary.